plugins/powerpc/powerpc_arith.ml

```diff
@@ -0,0 +1,420 @@
+
```

Comment:
that's strange modularization, it sounds like that the addition is not a part of arithmetics :)